ProcessList.c

```diff
@@ -655,7 +669,7 @@ void ProcessList_scan(ProcessList* this, bool pauseProcessUpdate) {
             p->tombStampMs = this->monotonicMs + 1000 * this->settings->highlightDelaySecs;
          } else {
             // immediately remove
-            ProcessList_remove(this, p);
+            ProcessList_removeIndex(this, p, i);
```

Comment:
This is still quadratic for large numbers of processes …

Reply:
Fixed now that we delay compacting the vector until we're done with all deletes in `ProcessList_scan`.